.gitmodules

```diff
@@ -42,7 +42,7 @@
 	url = https://github.com/ClickHouse/boost
 [submodule "contrib/base64"]
 	path = contrib/base64
-	url = https://github.com/ClickHouse/Turbo-Base64
+	url = https://github.com/aklomp/base64.git
```

Comment:
Git fails to checkout the new submodule in an existing repository with weird errors, no amount of `git submodule update` and `git reset --hard` helps. Repro (in case you like to try on an fresh, existing repo)

```sh
git remote add mkmkme https://github.com/mkmkme/ClickHouse.git
git fetch mkmkme
git switch --create mkmkme/aklomp-base64 mkmkme/mkmkme/aklomp-base64

git reset --hard && git clean --force -d
git submodule update --recursive --init
```

It won't be a problem in CI where the tree and all submodules are checked out from scratch but developers that rebase or merge this PR into their local repositories will complain (a lot). I think (not sure) that it is a problem that the existing `submodule` entry is reused. Unfortunately, someone reused the existing "base64" submodule already for "Turbo-base64" in the past already :-(

So, as as fix, could you try to
- make a commit which moves the existing Turbo64 submodule labeled as "base64" to a submodule labeled "Turbo64"?
- make a commit with the content of this PR what adds "base64" as a new submodule?

Reply:
Yeah, this might help.

There's also a solution of running `git submodule sync --recursive`. This is the command needed for syncing the submodule URLs.

`base64` had already been changed before, and it was also done within a commit: https://github.com/ClickHouse/ClickHouse/pull/8444/commits/063682ef9c20aeffd7aaa9d642ef263cd5f1b8e6

However, I can do it in two steps, if it makes the development easier :)

Reply:
Actually this might not work as well, because the changes will be applied all at once, not one-by-one.

How about I put the new library into `aklomp-base64` (and `aklomp-base64-cmake` respectively) instead, so there is no ambiguity in the future?

Reply:
Sounds good, let's go for `aklomp-base64'.

Reply:
I think I made a mistake of leaving the submodule name intact. I used `git rm contrib/aklomp-base64` and re-added it via `git submodule add`. Let's see if it helps. I'll also double-check this on my other clone of CH

Reply:
For my clone seems to have worked. I had to invoke `git submodule update --init`, but AFAU it's fine when there's a new submodule?

Reply:
I can confirm that it works with a new submodule.